ArcGISRuntimeSDKQt_CppSamples/Maps/CreateDynamicBasemapGallery/CreateDynamicBasemapGallery.qml

```diff
@@ -0,0 +1,163 @@
+                            required property string styleName
+                            required property string previewImageUrl
+                            onClicked: {
+                                model.updateSelectedStyle(styleName);
+                            }
+                            clip: true
+                            Column {
+                                spacing: 5
+                                padding: 10
+                                Text {
+                                    font.bold: true
+                                    font.pixelSize: 12
+                                    text: styleName + ":"
+                                }
+                                Image {
+                                    source: previewImageUrl
+                                    fillMode: Image.PreserveAspectFit
+                                    width: basemapView.cellWidth - 25
+                                }
+                            }
+                        }
+                    }
+
+                    ColumnLayout {
+                        Layout.alignment: Qt.AlignHCenter
+
+                        Text {
+                            Layout.preferredHeight: 15
+                            font.underline: true
+                            text: "Language Strategy:"
+                        }
+                        ComboBox {
+                            id: languageStrategyOptions
+                            Layout.preferredHeight: 30
+                            Layout.preferredWidth: 200
+                            model: model.languageStrategies
+                            enabled: model.languageStrategies.length !== 0
+                        }
+                        Text {
+                            Layout.preferredHeight: 15
+                            font.underline: true
+                            text: "Language:"
+                        }
+                        ComboBox {
+                            id: languages
+                            Layout.preferredHeight: 30
+                            Layout.preferredWidth: 200
+                            model: model.languages
+                            enabled: model.languages.length !== 0
+                        }
+                        Text {
+                            Layout.preferredHeight: 15
+                            font.underline: true
+                            text: "Worldview:"
+                        }
+                        ComboBox {
+                            id: worldviews
+                            Layout.preferredHeight: 30
+                            Layout.preferredWidth: 200
+                            model: model.worldviews
+                            enabled: model.worldviews.length !== 0
+                        }
+                        Button {
+                            height: 50
+                            text: "Load"
```

Comment:
I'd maybe try setting `Layout.preferredWidth: 200` so it matches the combo box width